specs/capri/Monitoring Design.md

```diff
@@ -0,0 +1,742 @@
+---
+title: "# Telemetry Design"
+---
```

Comment:
This looks weird thru "View file".  Just change to "# Telemetry Design"

Reply:
fixed